components/tabs/src/schema.rs

```diff
@@ -67,4 +102,47 @@ mod tests {
         conn.execute_batch(CREATE_SCHEMA_SQL)
             .expect("should allow running twice");
     }
+
+    #[test]
+    fn test_tabs_meta() {
+        let mut db = TabsStorage::new_with_mem_path("test");
+        //let conn = db.open_or_create().unwrap();
```

Comment:
nit - kill this line.

Also, consider moving these tests to db.rs (ie, where the meta functions being tested are actually defined). Ideally tests here would limit themselves to the schema - eg, manually selecting/inserting into the tab makes sense here.